datamover/pkg/drivers/https/prepare.go

```diff
@@ -244,7 +244,7 @@ func getHwObjs(ctx context.Context, conn *pb.Connector, filt *pb.Filter,
 		return nil, err
 	}
 	for i := 0; i < len(objs); i++ {
-		obj := osdss3.Object{Size: objs[i].Size, ObjectKey: objs[i].Key, Backend: ""}
+		obj := osdss3.Object{Size: objs[i].Size, ObjectKey: objs[i].Key, Location: ""}
```

Comment:
what is the difference, in example, between backend and location and why is it relevant now ?

Reply:
See above please.